Documentation/ApiOverview/SiteHandling/WriteCustomErrorHandler.rst

```diff
@@ -66,8 +66,18 @@ The ErrorHandler class:
    use TYPO3\CMS\Core\Error\PageErrorHandler\PageErrorHandlerInterface;
    use TYPO3\CMS\Core\Http\RedirectResponse;
 
-   class ErrorHandling implements PageErrorHandlerInterface 
+   class ErrorHandling implements PageErrorHandlerInterface
    {
+       protected int $statusCode;
+       protected array $errorHandlerConfiguration;
```

Comment:
Since we still support PHP < 7.4 these type hints should be done as annotation instead.

Reply:
Not used to that anymore ;) fixed it.